### DISCUSSION TITLE
combine mouse click and hover actions together

### DISCUSSION THREAD

**Discussion by cyranoDeCasa** [Tallyb/cucumber-playwright#586] [Q&A]
Hello,

I am looking for" how we can combine mouse click and hover actions together in playwright with typescript"
could you please help me for that?

after selecting the Zoom-in button, i would like to maintain the left click and move the cursor over an image to do a zoom-in

Thank you !

**Comment by Tallyb** [comment]
I think this is appropriate for discussing on the playwright discord.
